src/components/Card.vue

```diff
@@ -18,21 +21,48 @@
         </el-tooltip>
       </div>
       <div v-if="showCardDetails" class="details">
-        <p>
+        <p v-if="!data.hideType">
           <b>{{ data.type }}</b>
         </p>
-        <el-tooltip :content="data.title" placement="top">
-          <p class="title">
-            {{ data.title }}
-          </p>
-        </el-tooltip>
-        <el-button @click.prevent="openLinkInNewTab"> View {{ data.type }}</el-button>
+        <el-popover ref="galleryPopover" :content="data.title" placement="top" trigger="hover" popper-class="gallery-popper" />
+        <p v-if="!data.hideTitle" class="title" v-popover:galleryPopover>
+          {{ data.title }}
+        </p>
+        <el-button class="button" @click.prevent="cardClicked"> View {{ data.type }}</el-button>
       </div>
     </div>
   </el-card>
 </template>
 
 <script>
+// import { SvgIcon } from '@abi-software/svg-sprite'
+import axios from 'axios'
+import Vue from 'vue'
+import { Button, Card, Popover } from 'element-ui'
+Vue.use(Button)
+Vue.use(Card)
+Vue.use(Popover)
+
+function isValidHttpUrl(string) {
+  let url = undefined
+
+  try {
+    url = new URL(string)
+  } catch (_) {
+    return false
+  }
+  return url.protocol === 'http:' || url.protocol === 'https:'
+}
+
+const getRequest = async (url, params, timeout) => {
+  return await axios({
+    method: 'get',
+    url,
+    params,
+    timeout,
+  })
+}
+
```

Comment:
This is duplicated code from GalleryHelpers.

Reply:
Ok, I will switch to use that instead.